packages/camera/camera_avfoundation/ios/Classes/FLTCam.m

```diff
@@ -162,7 +168,9 @@ - (instancetype)initWithCameraName:(NSString *)cameraName
   _motionManager = [[CMMotionManager alloc] init];
   [_motionManager startAccelerometerUpdates];
 
-  [self setCaptureSessionPreset:_resolutionPreset];
+  if (![self setCaptureSessionPreset:_resolutionPreset withError:error]) {
```

Comment:
This also technically a behavioral change in that now the error actually gets returned instead of us having an unhandled `@throw` of an `NSError`, which... I don't even actually know what that would do. Crash the app probably?

But as above, this should not actually be a thing that can happen as far as I can tell.